t/32-openqa_client.t

```diff
@@ -77,11 +79,11 @@ subtest 'upload private assets' => sub {
     my ($local_prepare, $chunk_prepare);
     my $local_prepare_cb = $t->ua->upload->on('upload_local.prepare' => sub { $local_prepare++ });
     my $chunk_prepare_cb = $t->ua->upload->on('upload_chunk.prepare' => sub { $chunk_prepare++ });
-    eval {
+    lives_ok {
         $t->ua->upload->asset(
             99963 => {chunk_size => $chunk_size, file => $filename, name => 'hdd_image3.qcow2', asset => 'private'});
-    };
-    ok !$@, 'No upload errors' or die explain $@;
+    }
+    'No upload errors' or die explain $@;
```

Comment:
Here you kept the die, but I suggested to use `BAIL_OUT`.
So I suggest to use `BAIL_OUT` in all places where previously was a `die`.
I just don't want to do a manual suggestion in every place. You are expected to look at similar occurrences yourself.

Reply:
from https://github.com/os-autoinst/openQA/pull/6263#discussion_r1984879191 the suggestion was clearly about two lines. I assumed that there is no reason to modify the rest due to https://github.com/os-autoinst/openQA/pull/6263#discussion_r1984170905

Reply:
https://github.com/os-autoinst/openQA/pull/6274